client/rule.go

```diff
@@ -254,6 +254,21 @@ func (rc *RuleClient) Run() error {
 
 		if len(pts) > 0 {
 			active, changed, err = rc.ruleProcessPoints(id, pts)
+			if rc.config.Disabled {
```

Comment:
I don't think this is the right place for this logic. 

in the main for/select loop, under the `case pts := <-rc.newPoints:` case statement, you can check if it is a disabled point, and take appropriate action there. If the node ID = the top level rule node ID, then:

- if the disabled is false, simply run the rule
- if disabled is true, then run the ruleInactiveActions, and set the active flag to false.

If the node ID != top level rule ID, then it is a sub node (action or condition) where disabled is changed. It may be sufficient to simply run the rule -- I'm not sure -- would need to simulate. Ideally tests would cover all this, but this could be a lot of tests.